webapp/package.json

```
@@ -26,56 +26,59 @@
   "dependencies": {
     "@azure/msal-node": "^2.16.2",
     "@types/ioredis": "^4.26.7",
-    "@types/jest": "^29.5.14",
     "axios": "^1.9.0",
     "cookie": "^1.0.2",
     "google-libphonenumber": "3.2.32",
     "govuk-frontend": "^4.10.0",
     "ioredis": "^4.27.3",
     "isodate": "^0.1.4",
-    "jest": "^29.7.0",
     "lodash": "^4.17.21",
-    "next": "^14.2.29",
+    "next": "15.3.4",
```

Can we let these take non breaking compatible releases?

E.g.
```suggestion
    "next": "^15.3.4",
```